Update/omot_017.txt

```diff
@@ -2744,13 +2744,13 @@ void main()
 //r明日は村を総出で、鉄平を捜索するらしい。…警察に引き渡す前に、せめて目にもの見せなければ気がすまない、ということだろうか。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "明日は村を総出で、鉄平を捜索するらしい。…警察に引き渡す前に、せめて目にもの見せなければ気がすまない、ということだろうか。",
-			NULL, "Apparently, they're going to head out of the village tomorrow to search for Teppei. I can't imagine what they might do to him before handing him over to the police.", Line_Normal);
+			NULL, "Apparently, all of them are going to search this village for Teppei tomorrow. I guess it's because they have to get even with him before handing him over to the police.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Apparently, all of them are going to search this village for Teppei tomorrow. I guess it's because they want to get even with him before handing him over to the police.", Line_Normal);
```